### PR TITLE
Update js_jquery_toggle and fix typos

### PR DESCRIPTION
**Pull request by Gehock**
# Description

Updating `js_jquery_toggle` ensures that static files are correctly collected from the package. Additionally, a small typo fixes in the installer script and documentation have been included.

## Testing

The changes have been tested manually on the A+ test server.

# Is it [Done](https://wiki.aalto.fi/display/EDIT/Definition+of+Done)?

- [x] I (developer) have tested the changes manually
- [x] Reviewer has finished the code review
- [x] After the review, the developer has made changes accordingly

### CODE REVIEW

File: jutut/local_settings.example.py [hunk 1]
```diff
@@ -8,6 +8,9 @@
 #SERVER_EMAIL = 'root@example.com' # Error message sender
 #DEFAULT_FROM_EMAIL = 'webmaster@example.com' # Other email sender
 
+# Make sure this points to the correct virtual environment location
+STATICFILES_DIRS += join(BASE_DIR, '../venv/js_jquery_toggle/static'),
```
Comment (raphendyr) [inline jutut/local_settings.example.py:12]: This shouldn't be required. The static file is found as it's part of the js_jquery_toggle django app. Ensure that:
1. You have installed the django wrapped version of js_jquery_toggle
2. `INSTALLED_APPS` includes the django part of the js_jquery_toggle (it's possible it was included as a dependency using raphendyr-django-essentials featyre, I don't remember)
3. Ensure the js_jquery_toggles, setup.py installs static files correctly (packages them to the wheel and then installs those to correct paths)

File: scripts/install.py [hunk 2]
```diff
@@ -694,7 +694,7 @@ def do_init():
             dest=OPTS.dest,
             conf=conf,
             conf_arg=('-c {}'.format(conf) if OPTS.config else ''),
-            dconf=OPTS.dest / 'jutut' / 'local_settings,py',
+            dconf=OPTS.dest / 'jutut' / 'local_settings.py',
```
Comment (raphendyr) [inline scripts/install.py:697]: Interesting that I never noticed that.

File: jutut/local_settings.example.py (hunk 1 above)
Comment (raphendyr) [reply jutut/local_settings.example.py:12]: For the future, wrapping js libraries on django apps isn't a good idea, instead they should be pulled as part of the CI/CD pipeline and packaging (see A+ drone configs). However, none of the Django apps in the project include packaging :cry:

Comment (raphendyr) [reply jutut/local_settings.example.py:12]: Are you using a version, which is after this commit: https://github.com/Aalto-LeTech/js-jquery-toggle/commit/e100529371a6e3dba2fc22a6a57fc80f4d1ededf

Comment (raphendyr) [reply jutut/local_settings.example.py:12]: Yes, I think that is the issue. Update the requirements.txt with the 1.1.1 version of the lib.

```patch
-git+https://github.com/Aalto-LeTech/js-jquery-toggle.git@1.1.0#egg=js-jquery-toggle-django==1.1.0&subdirectory=django
+git+https://github.com/Aalto-LeTech/js-jquery-toggle.git@1.1.2#egg=js-jquery-toggle-django==1.1.2&subdirectory=django
```

Comment (Gehock) [reply jutut/local_settings.example.py]: That's definitely a lot more elegant solution!

**Review by Mankro — APPROVED** [review]
@Gehock nice little fix! @raphendyr thanks for helping out with the static files!

File: requirements.txt [hunk 3]
```diff
@@ -4,7 +4,7 @@ Jinja2 ~= 2.10.3
 django-jinja==2.3.1
 django-bootstrap-form==3.2.1
 django-jinja-bootstrap-form ~= 4.2.3
-git+https://github.com/Aalto-LeTech/js-jquery-toggle.git@1.1.0#egg=js-jquery-toggle-django==1.1.0&subdirectory=django
+git+https://github.com/Aalto-LeTech/js-jquery-toggle.git@1.1.2#egg=js-jquery-toggle-django==1.1.0&subdirectory=django
```
Comment (raphendyr) [inline requirements.txt:7]: A mistake in this line. There is different tag compared to python package version requirement. Tag is `1.1.2`, but package requirement is `1.1.0`. I'm not sure how `pip` behaves with this.

Comment (Mankro) [reply requirements.txt:7]: Is this the correct line?
`git+https://github.com/Aalto-LeTech/js-jquery-toggle.git@1.1.2#egg=js-jquery-toggle-django==1.1.2&subdirectory=django`
I will change this straight away myself.

Comment (raphendyr) [reply requirements.txt:7]: Looks good. [The syntax](https://pip.pypa.io/en/stable/reference/pip_install/#vcs-support) is

```
vcs+protocol://repo_url/#egg=pkg&subdirectory=pkg_dir
```

Where `repo_url` is git reference, i.e., here `https://github.com/Aalto-LeTech/js-jquery-toggle.git@1.1.2` including the tag (as we want specific version from the git and not the "master").

And the `#egg=` contains the "normal" requirements line, i.e.:

> The “project name” component of the URL suffix egg=<project name> is used by pip in its dependency logic to identify the project prior to pip downloading and analyzing the metadata.

**Pull request merged**